Source/UIKit/UILabel.swift

```diff
@@ -14,4 +14,8 @@ extension UILabel {
     public var rex_text: MutableProperty<String> {
         return rex_stringProperty("text")
     }
+    
+    public var rex_textColor: MutableProperty<UIColor> {
+        return rex_valueProperty(&textColor, { [weak self] in (self?.textColor)! }, { [weak self] in self?.textColor = $0 })
```

Comment:
The `MutableProperty` shouldn't outlive `self` though, right? Ideally it just completes if `UILabel` is deallocated. For that reason I'd actually consider using `unowned` instead, to better represent the ownership model in place here.